src/rucksack-2-disable-glob.patch

```diff
@@ -0,0 +1,62 @@
+diff --git a/src/main.c b/src/main.c
+index 24583e8..19196d2 100644
+--- a/src/main.c
++++ b/src/main.c
+@@ -11,7 +11,6 @@
+ #include <time.h>
+ #include <unistd.h>
+ #include <assert.h>
+-#include <glob.h>
```

Comment:
Are you expecting `HAVE_GLOB_T` to be defined on a POSIX system? I just tried it on linux and `HAVE_GLOB_T` is not defined.